README.md

```diff
@@ -278,3 +278,33 @@ The relevant pages can be checked to adopt a similar design.
 2. Find the color section you wish the change (it is commented)
 3. Change the variable here only.
 4. Do not change the code below the variables unless you wish to add or remove a color instead of changing a color.
+
+### How to add new icons?
+
+1. **Install FontForge:**
+   * Download and install FontForge from [FontForge's official website](https://fontforge.org/en-US/).
+
+2. **Create or Prepare Your Icon:**
+   * Create your icon in a vector graphics editor and save it as an SVG file.
+   * Ensure that your icon is properly scaled and fits well within the boundaries of a single glyph.
```

Comment:
I think a link here could help a person understand what you mean by "properly scaled"